commcare_connect/opportunity/models.py

```diff
@@ -268,7 +271,7 @@ def last_visit_date(self):
 
     @property
     def total_paid(self):
-        return Payment.objects.filter(opportunity_access=self).aggregate(total=Sum("amount")).get("total", 0)
+        return Payment.objects.filter(opportunity_access=self).aggregate(total=Sum("amount")).get("total", 0) or 0
```

Comment:
When does the `or` trigger here. It looks like the other thing should default to zero?

Reply:
The thing should default to 0 but it was None in cases where no Payment objects existed, this prevents this method from returning None.